### PR TITLE
Added contributing.md file

### PR DESCRIPTION
**Pull request by shraddhavp**
added the contributing.md file closes issue #40

### CODE REVIEW

**Comment by shraddhavp** [comment]
@anushbhatia  sir  created PR for contributing.md file please  check it and let me know for any changes

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#Contributing
```
Comment (anushbhatia) [inline CONTRIBUTING.md:1]: add space between # and contribution

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Pull Request process
+
+1.Clone the repository 
+2.Install all the needed dependencies 
+3.Get an issue assigned to you by the developers
+4.Raise a PR with the issue number and  a description of changes done  
+5.Request the reviewers to merge the issue for you 
+
+##Code of Conduct
```
Comment (anushbhatia) [inline CONTRIBUTING.md:13]: we already have code of conduct for this

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other standard  method with the owners of this repository before making a change.
+
+## Pull Request process
```
Comment (anushbhatia) [inline CONTRIBUTING.md:5]: https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/CONTRIBUTING.md
add code snippets to the file.
Grading sheet
you  can add gif's too

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+4.Raise a PR with the issue number and  a description of changes done  
+5.Request the reviewers to merge the issue for you 
+
+##Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our Standards
```
Comment (anushbhatia) [inline CONTRIBUTING.md:19]: isn't this part of code of conduct why should we mention this in our contribute.md

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* Other conduct which could reasonably be considered inappropriate in a
+ professional setting
+
+### Our Responsibilities
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+### Scope
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
```
Comment (anushbhatia) [inline CONTRIBUTING.md:47]: this is not code of conduct please take care and rectify this file

**Review by anushbhatia — CHANGES_REQUESTED** [review]

**Comment by shraddhavp** [comment]
ok I will do it

**Pull request closed without merging**